app/templates/compare_revisions.html

```diff
@@ -63,27 +63,19 @@
     <div class="grid-row">
       <div class="column-two-thirds page-section">
         <p>
-          {% if all_update_events is none %}
-            Multiple
+          {% if latest_update_events %}
+            {% if min_number_of_users_who_made_edits > 1 %}
+              More than one user has edited this service. The last user to edit this page was
+            {% endif %}
+            {{ latest_update_events.0.user }}
+            {% if min_number_of_users_who_made_edits == 1 %}
+              made {{ (all_update_events or (oldest_update_events+latest_update_events))|length }}
+              {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") }}
+            {% endif %}
+            on {{ latest_update_events[0].createdAt|dateformat -}}
           {% else %}
             {{ all_update_events|length }}
-          {% endif %}
-          unapproved {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") }}
-          {%- if latest_update_events %}
-            {% if all_update_events is not none %}
-              by
-              {% if n_editing_users_min == 1 %}
-                {{ latest_update_events.0.user }}
-              {% else %}
-                {{ n_editing_users_min }} users
-              {% endif %}
-            {% endif %}
-            {% if latest_update_events[0].createdAt|dateformat == oldest_update_events[-1].createdAt|dateformat %}
-              on {{ latest_update_events[0].createdAt|dateformat }}
-            {%- else %}
-              between {{ oldest_update_events[-1].createdAt|dateformat }}
-              and {{ latest_update_events[0].createdAt|dateformat }}
-            {%- endif -%}
+            unapproved {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") -}}
```

Comment:
The way this code is restructured now it looks like this "else" part can only ever be reached if there are no edits to the service, so it could just say `0 unapproved edits` here rather than `{{ all_update_events|length }} unapproved {{ pluralize((all_update_events or (oldest_update_events+latest_update_events))|length, "edit", "edits") -}}`

Could even be a nicer sentence like `This service has no unapproved edits` or something.

Or have I missed some case where we might have no `latest_update_events` but still some edits to report?

Reply:
Good point, I don't see how we could have no latest_update_edits but have some other ones.